### PR TITLE
[MFTF] Refactoring AdminCheckDashboardWithChartsTest

### LINKED ISSUE #31523: [Issue] [MFTF] Refactoring AdminCheckDashboardWithChartsTest
This issue is automatically created based on existing pull request: magento/magento2#31451: [MFTF] Refactoring AdminCheckDashboardWithChartsTest

---------
### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Go to Admin Dashboard
2. Grab orders quantity
3. Place/invoice/ship an order
4. Reload the Dashboard
5. Check the display
6. Check if orders quantity increased

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Go to Admin Dashboard
2. Grab orders quantity
3. Place/invoice/ship an order
4. Reload the Dashboard
5. Check the display
6. Check if orders quantity increased

### Resolved issues:
1. [x] resolves magento/magento2#31523: [MFTF] Refactoring AdminCheckDashboardWithChartsTest

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by sidolov** [comment]
@magento create issue

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminDashboardDisplayIsProperActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminDashboardDisplayIsProperActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminDashboardDisplayIsProperActionGroup.xml]: I would suggest avoiding something like "proper", "correct" in the naming. Using these keywords might confuse on an attempt to understand what is "proper". I would recommend something like `AssertAdminDashboardDisplayedWithNoErrorsActionGroup`. 

Thanks

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @AnnaAPak. Good job! Please, check my comments below.

File: app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminDashboardDisplayIsProperActionGroup.xml (hunk 1 above)
Comment (engcom-Hotel) [reply app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminDashboardDisplayIsProperActionGroup.xml]: Has been renamed.

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Unit Tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by engcom-Hotel** [comment]
@magento run Functional Tests B2B

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-8822](https://jira.corp.magento.com/browse/ENGCOM-8822) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**